src/libcollections/vec_map.rs

```diff
@@ -1140,6 +1213,52 @@ mod test_map {
     }
 
     #[test]
+    fn test_append() {
+        let mut a = VecMap::new();
+        a.insert(1, "a");
+        a.insert(2, "b");
+        a.insert(3, "c");
+
+        let mut b = VecMap::new();
+        b.insert(3, "d");  // Overwrite element from a
+        b.insert(4, "e");
+        b.insert(5, "f");
```

Comment:
For reference, you can collect an iterator of tuples. e.g. [(1, "a"), (2, "b"), (3, "c")].